### PR TITLE
Seed/GBR-04A OrgMixin Cycle 

### PR DESCRIPTION
**Pull request by fablet**
#### Definition of Done:
- [ ] Is there appropriate test coverage? (e.g. ChefSpec, Mocha/Chai, Python, etc.)
- [ ] Does this PR require a Selenium test? (e.g. Browser-specific bugs or complicated UI bugs)
- [ ] Does this PR require a regression test? All fixes require a regression test.
- [ ] Does this add new dependencies? If so, does PIP, npm, bower requirements need to be updated?

### CODE REVIEW

File: seed/filtersets.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        name = "{} Calendar Year".format(value)
+        cycles = queryset.filter(name__contains=name)
+        if not cycles:
+            start = datetime(int(value), 1, 1)
+            end = start + relativedelta(years=1) - relativedelta(seconds=1)
+
+            # to eliminate the question of timezone saved in vs timezone
+            # queried from, start and end dates are nudged in by the max
+            # possible time difference between 2 servers
+            start = start + relativedelta(hours=max_time_diff)
```
Comment (tallus) [inline seed/filtersets.py]: This is just to select the relevant cycle?

File: seed/views/cycles.py [hunk 2]
```diff
@@ -1,192 +1,151 @@
 # !/usr/bin/env python
 # encoding: utf-8
 """
-:copyright (c) 2014 - 2016, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
-:author Paul Munday<paul@paulmunday.net>
+:copyright (c) 2014 - 2016, The Regents of the University of California,
+through Lawrence Berkeley National Laboratory (subject to receipt of any
+required approvals from the U.S. Department of Energy) and contributors.
+All rights reserved.  # NOQA
+:author Fable Turas <fable@raintechpdx.com>
```
Comment (tallus) [inline seed/views/cycles.py]: you should add yourself to Authors without removing existing (even though this looks like a complete rewrite) so contributors can be tracked over time

**Comment by tallus** [comment]
Whats the realtionship between this and No 4?

File: seed/filtersets.py (hunk 1 above)
Comment (fablet) [inline seed/filtersets.py]: yes.  it doesn't save those start or end dates anywhere, just creates a filter range

**Comment by fablet** [comment]
No 4 shouldn't be included in the branches we want to merge upstream at this time.  The work on datasets and labels wasn't strictly necessary for our purposes, but the work on cycles is.  If we wish to submit the datasets and lables as well, I will need to cut a new branch that pulls the pertinent files out of No 4 since this branch supersedes all work from that branch related to cycles

File: seed/views/cycles.py (hunk 2 above)
Comment (fablet) [inline seed/views/cycles.py]: It is a complete re-write, but that makes better sense than just doing the overwrite. Sorry about that. Fixed

**Pull request closed without merging**